ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2

```diff
@@ -1,5 +1,7 @@
 {# Leaf server port-channels #}
 port_channel_interfaces:
+{# This must be assigned outside the loop as switch is overridden in the loop sinner scope and can no longer be referenced #}
+{% set support_storm_control = switch.platform_settings.feature_support.interface_storm_control is not arista.avd.defined(false) %}
```

Comment:
I would use true as default to not break existing deployment unless we explicitly activate it for default platform type.

Reply:
I believe this logic is correct, but instead of setting a boolean up here like this, I think we should just change the logic in line 96 to:
`if adapter_storm_control is arista.avd.defined and switch.platform_settings.feature_support.interface_storm_control is not arista.avd.defined(false)`

Reply:
fixed